src/PlayFabBuddy.Infrastructure/Adapter/PlayFab/PlayerAccountAdapter.cs

```diff
@@ -51,10 +51,9 @@ public async Task<MasterPlayerAccountEntity> LoginWithCustomId(string customId)
          *  
          */
         var loginResult = await PlayFabClientAPI.LoginWithCustomIDAsync(request);
-        var mainAccount = new MasterPlayerAccountAdapter(loginResult.Result.AuthenticationContext.PlayFabId);
-        var titleAccount =
-            new TitlePlayerAccountAdapter(loginResult.Result.AuthenticationContext.EntityId, mainAccount.MainAccount);
 
-        return mainAccount.MainAccount;
+        var aggregate = new PlayerAccountAggregate(loginResult.Result.AuthenticationContext.PlayFabId, loginResult.Result.AuthenticationContext.EntityId);
```

Comment:
Why create an Aggregate if you're not returning it?

Either return teh aggregate, or just create the `MasterPlayerAccountEntity`.

I would favour returni9ng teh aggregate.